### PR TITLE
[Project] Added a supported for Visual Attention Network (VAN)

### PR DESCRIPTION
**Pull request by tang576225574**
## Motivation
The original version of Visual Attention Network (VAN) can be found from
https://github.com/Visual-Attention-Network/VAN-Segmentation
添加Visual Attention Network (VAN)的支持。

## Modification
added a floder mmsegmentation/projects/van/
added 13 configs totally and aligned performance basically.
只增加了一个文件夹，共增加13个配置文件，基本对齐性能（没有全部跑）。

## Use cases (Optional)
Before running, you may need to download the pretrain model from https://cloud.tsinghua.edu.cn/d/0100f0cea37d41ba8d08/
and then move them to the folder mmsegmentation/pretrained/, i.e. "mmsegmentation/pretrained/van_b2.pth".
After that, run the following command:
    cd mmsegmentation
    bash tools/dist_train.sh projects/van/configs/van/van-b2_pre1k_upernet_4xb2-160k_ade20k-512x512.py 4

### CODE REVIEW

**Comment by csatsurnh** [comment]
Hi @tang576225574, thank you for your nice pr, but please sign the CLA first before we can review on it. Besides, can you also fix the lint please, thank you so much.

**Comment by tang576225574** [comment]
> Hi @tang576225574, thank you for your nice pr, but please sign the CLA first before we can review on it. Besides, can you also fix the lint please, thank you so much.

ok, I will fix it today.

**Comment by tang576225574** [comment]
> Hi @tang576225574, thank you for your nice pr, but please sign the CLA first before we can review on it. Besides, can you also fix the lint please, thank you so much.

lint had been fixed.

**Comment by xiexinch** [comment]
Hi @tang576225574,
Thanks for your contribution!
Could you write ad README for this project?

**Comment by tang576225574** [comment]
> Hi @tang576225574, Thanks for your contribution! Could you write a README for this project?

of course! I will  write a README in the next commit, about 3 days.

File: projects/van/configs/_base_/models/van_fpn.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+data_preprocessor = dict(
+    type='SegDataPreProcessor',
+    mean=[123.675, 116.28, 103.53],
+    std=[58.395, 57.12, 57.375],
+    bgr_to_rgb=True,
+    pad_val=0,
+    seg_pad_val=255,
+    size=(512, 512),
+    test_cfg=dict(size_divisor=32))
```
[the remark this answers is not in the document]
Comment (xiexinch) [reply projects/van/configs/_base_/models/van_fpn.py:12]: 可以在你的project的config目录下新建一个ade20k.py，重新写一个test_pipeline。

**Comment by tang576225574** [comment]
Fixed！

**Review by xiexinch — APPROVED** [review]

**Pull request merged**